corehq/apps/hqadmin/reporting/reports.py

```diff
@@ -365,8 +365,7 @@ def get_active_supply_points_data(domains, datespan, interval):
         num_active_supply_points = (CaseES()
                 .domain(domains)
                 .case_type(SUPPLY_POINT_CASE_TYPE)
-                .active_in_range(gte=start_date, lte=end_date)
-                .size(0)).run().hits
+                .active_in_range(gte=start_date, lte=end_date)).run().hits
```

Comment:
That API is a little confusing, I agree.  I initially called it `hits` because that's the terminology that elasticsearch uses, but I'd support adding an alias called `docs`, which is more in line with our terminology.